app/static/src/components/Shopcart.vue

```diff
@@ -18,7 +18,7 @@
 
             <div class="col-sm">
               <input type="number"
-                :id="`product-${product.id}-quantity`"
+                :id="`shopcart-product-${product.id}-quantity`"
```

Comment:
I had the same problem so changed this to a h6 instead of an input, which does not works as an input, and changed the id in products-id-quantity to  products-id-slect, so you'll probably have a conflict here.

Reply:
@radhikamattoo discard this, I just saw your review.